spec/polymorphic_integer_type_spec.rb

```diff
@@ -193,6 +194,17 @@
     end
   end
 
+  context "When a link is given polymorphic record via a non-integer type association" do
```

Comment:
This spec mimic the behavior we're seeing in the main rma repo and fails on the main branch, but passes on this branch